samples/net/openthread/coap_server/README.rst

```diff
@@ -20,24 +21,49 @@ Requirements
 User interface
 **************
 
+The following board buttons are used by this sample:
+
+Button 4:
+    Pressing results in etering pairing mode for the limited amount of time.
+
+LED 3:
+    Indicates pairing mode enabled when blinking.
+
+LED 4:
+    Turned on and off from the CoAP Clients.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/net/openthread/coap_server`
+
+.. include:: /includes/build_and_run.txt
+
 Testing
```

Comment:
See my comments to the client's Testing section and apply the relevant changes here as well.